README.md

```diff
@@ -1,101 +1,232 @@
+
 # Lean 4 VSCode Extension
 This extension provides VSCode editor support for the Lean 4 programming language.
 It is based on [vscode-lean](https://github.com/leanprover/vscode-lean).
 
-## Running the client
-1. Install a Lean 4 nightly build as described [in the documentation](https://leanprover.github.io/lean4/doc/setup.html).
-2. Install the extension from the [marketplace](https://marketplace.visualstudio.com/items?itemName=leanprover.lean4).
-3. Create a Lean 4 project using [leanpkg](https://leanprover.github.io/lean4/doc/setup.html#leanpkg) and open the project folder with VSCode. If you have a global installation of Lean 4, e.g. using `elan default leanprover/lean4:nightly` as described in the documentation above, you can also just open single `.lean` files independent of a project folder.
-4. Open a `.lean` file and type in `#eval 1`. The extension should display a blue underline below `#eval`. Upon hovering over it, a hover panel reporting the result of the evaluation should pop up. When hovering over the `1`, a hover panel displaying the type of `1` should pop up.
-
-If no blue underline is displayed, make sure that Lean 4 is installed correctly by running `lean --version` in your project folder. The VSCode extension will not activate if it cannot find the `lean` command or `lean --version` reveals that Lean 3 is installed instead.
+## Installing the extension and Lean 4
+1. Install the extension from the [marketplace](https://marketplace.visualstudio.com/items?itemName=leanprover.lean4).
+1. Open a folder containing .lean code using VS Code.  It is recommended that you open a folder that is configured using [lake](https://github.com/leanprover/lake) which is a lean package configuration tool.
```

Comment:
Is the all-`1` numbering intentional? It does display correctly (1,2,3,..) on GitHub.

Reply:
```suggestion
1. Open a folder containing Lean code using VS Code.  It is recommended that you open a folder that is configured using [Lake](https://github.com/leanprover/lake) which is a Lean package configuration tool.
```

Reply:
Fixed and yes the "1." numbering trick makes it easier to re-order items in the list - the markdown processor numbers the items correctly in the generated HTML.  The vscode markdown viewer also does this correctly.